frontend/app/templates/admin.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="container stackable doubling centered grid">
+    <div class="ui hidden divider"></div>
+    {{#if session.isAuthenticated}}
+        <h1>Admin</h1>
```

Comment:
This is not the change I am requesting, it should read route the page according to session before loading the website.

Reply:
Do the changes that are requested. @vaibhavsingh97